Client/Frontend/PasswordManagement/Cells/PasswordManagerTableViewCell.swift

```diff
@@ -117,7 +119,9 @@ class PasswordManagerTableViewCell: ThemedTableViewCell {
     override func applyTheme(theme: Theme) {
         super.applyTheme(theme: theme)
         hostnameLabel.textColor = theme.colors.textPrimary
+        hostnameLabel.font =  DefaultDynamicFontHelper.preferredFont(withTextStyle: .title1, size: 16.0)
```

Comment:
Typically in the project, we set the font in the label declaration in this case you could replace lines 43 and 50 for each label to use `DefaultDynamicFontHelper.preferredFont`

Reply:
@yoanarios Thank you, should be fixed in 82779670b54706c826acc8656f11c94a4be46736